src/redux/state.js

```diff
@@ -69,10 +88,18 @@ const store = {
       case ACTION_TYPES.UPDATE_POST:
         this._updateArea(action.payload);
         break;
+      case ACTION_TYPES.ADD_NEW_MESSAGE:
+        this._addNewMessage();
+        break;
+      case ACTION_TYPES.UPDATE_NEW_MESSAGE:
+        this._updateNewMessage(action.payload);
+        break;
       default: this._renderFunc();
     }
   },
 
 };
 
-export default store;
+const actionCreater = (typeOf, text) => ({ type: typeOf, payload: text });
```

Comment:
Consider calling me for a live session regarding this functionality.

Reply:
At now actioncreator look like cool.thanks for call.It was usefull